src/main/java/org/jpmml/sparkml/bootstrap/Main.java

```diff
@@ -93,15 +94,17 @@ public void main(String... args) throws Exception {
 	private void run() throws Exception {
 		SparkConf sparkConf = new SparkConf();
 
-		try(JavaSparkContext sparkContext = new JavaSparkContext(sparkConf)){
-			SQLContext sqlContext = new SQLContext(sparkContext);
+		  SparkSession spark = SparkSession
+                .builder()
+                .appName("my app")
+                .getOrCreate();
 
-			DataFrameReader reader = sqlContext.read()
-				.format("com.databricks.spark.csv")
-				.option("header", "true")
-				.option("inferSchema", "true");
-
-			DataFrame dataFrame = reader.load(this.csvInput.getAbsolutePath());
+          Dataset<Row> dataFrame = spark.read()
+                .format("csv")
+                .option("header", "true")
+                .option("nullValue", "")
```

Comment:
should keep ``.option("inferSchema", "true")``

Otherwise everything is assumed to be a string, and the output pmml files include all of the possible values in the data dictionary`